packages/html/lib/jsx-runtime.js

```diff
@@ -49,7 +49,11 @@ const attrMap = {
     leafclassname: 'leafClassName',
 
     // Template
-    contenttemplate: 'contentTemplate'
+    contenttemplate: 'contentTemplate',
+
+    // ColorPalette
+    palette: 'palette',
```

Comment:
this might be not needed as `palette` is one word which output attribute is the same.